experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,212 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-disk-loss"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+    cloud_platform: "{{ lookup('env','CLOUD_PLATFORM') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    disk_name: "{{ lookup('env','DISK_NAME') }}"
+    device_name: "{{ lookup('env', 'DEVICE_NAME') }}"
+    node_name: "{{ lookup('env','NODE_NAME') }}"
+    project_id: "{{ lookup('env','PROJECT_ID') }}"
+    zone_name: "{{ lookup('env','ZONE_NAME') }}"
+    auxiliary_appinfo: "{{ lookup('env','AUXILIARY_APPINFO') }}"
```

Comment:
Do we also need ramp_time here? @ksatchit

Reply:
Addressed